ddl/main_test.go

```diff
@@ -62,6 +62,7 @@ func TestMain(m *testing.M) {
 		goleak.IgnoreTopFunction("github.com/golang/glog.(*loggingT).flushDaemon"),
 		goleak.IgnoreTopFunction("go.etcd.io/etcd/client/pkg/v3/logutil.(*MergeLogger).outputLoop"),
 		goleak.IgnoreTopFunction("github.com/tikv/client-go/v2/txnkv/transaction.keepAlive"),
+		goleak.IgnoreTopFunction("go.opencensus.io/stats/view.(*worker).start"),
```

Comment:
it can be stopped by ```view.Stop()```.

Reply:
It is kind of confusing to spread `view.Stop()` everywhere. Let's clean these up later.